csrc/scheduler/normalization_inner.h

```diff
@@ -65,17 +65,17 @@ class InnerPersistentKernelScheduler : public SchedulerEntry {
       const scheduler_utils::ReductionTvProperties& properties);
 };
 
-TORCH_CUDA_CU_API std::shared_ptr<ReductionParams> getInnerPersistentHeuristics(
```

Comment:
Ideally, the main branch should no longer have TORCH_CUDA_CU_API. but don't know why it is still there.

Reply:
Perhaps the branch was merged without integrating the most recent updates from the main branch.

Reply:
move the clean up to #920